zqd/api/api.go

```diff
@@ -119,9 +119,11 @@ type LogPostWarning struct {
 }
 
 type LogPostStatus struct {
-	Type string     `json:"type"`
-	Span *nano.Span `json:"span,omitempty"`
-	Size int64      `json:"size" unit:"bytes"`
+	Type         string     `json:"type"`
+	Span         *nano.Span `json:"span,omitempty"`
+	Size         int64      `json:"size" unit:"bytes"`
```

Comment:
This field is kind of confusing next to `Log*Size`. The brim client doesn't use it and it can easily find this by calling `GET /space/:space`. I think we should drop it.

Reply:
And the `Size` field doesn't appear to be updated until the very end.